app/windows/codesign.js

```diff
@@ -0,0 +1,166 @@
+const { exec } = require('child_process');
+const path = require('path');
+const os = require('os');
+const fs = require('fs');
+
+const SIGN_JSON_TEMPLATE = {
+  Version: '1.0.0',
+  DriEmail: ['joaquim.rocha@microsoft.com'],
```

Comment:
I'm not sure if this should be a personal email?

Reply:
This is the contact. I guess I can add hello at kinvolk.